style.css

```diff
@@ -9,4 +9,83 @@ box-sizing: border-box;
 body {
   font-size: 16px;
   margin: 0;
+  font-family: sans-serif;
+}
+.header {
+background: white;
+position: relative;
+}
+.top-image {
+width: 100%;
+height: 15rem;
+
+}
+.top-logo {
+  border-radius: 50%;
+  margin: 1rem;
```

Comment:
I would recommend not mixing position and `margin` since it can make things a bit harder to visualize why things are positioned where they are.

So to pull the image further off the left edge, use `left` instead of `margin`